webapp/channels/src/types/store/storage.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export type StorageItem<T = any> = {
```

Comment:
Ideally, this would default to `unknown` instead of `any`, but feel free to leave it as-is since this sort of typing thing seems like something that could end up taking days to solve 😅